common/scala/src/main/scala/whisk/core/containerpool/ContainerFactory.scala

```diff
@@ -31,7 +31,12 @@ case class ContainerArgsConfig(network: String,
                                dnsServers: Seq[String] = Seq.empty,
                                extraArgs: Map[String, Set[String]] = Map.empty)
 
-case class ContainerPoolConfig(numCore: Int, coreShare: Int) {
+case class ContainerPoolConfig(numCore: Int,
+                               coreShare: Int,
+                               maxConcurrent: Int = 1,
+                               concurrentPeekFactor: Double = 0.5) {
```

Comment:
Please don't define defaults in code. That will lead to multiple cases where defaults are applied which will rot over time. The ones in `application.conf` should be enough?

Reply:
yep 👍